lib/instrumentation/modules/@elastic/elasticsearch.js

```diff
@@ -70,7 +70,7 @@ module.exports = function (elasticsearch, agent, { version, enabled }) {
           if (result) {
             const currSpan = ins.currSpan()
             if (currSpan) {
-              diagResultFromSpan[currSpan] = result
+              diagResultFromSpan.set(currSpan, result)
```

Comment:
:embarrassment:

Reply:
it got by my review too so sign me up for the :embarrassment: newsletter.

Reply:
Wow, that's a pitfall if I have ever seen one.

Reply:
Dynamic languages for you. Not sure if TypeScript would have caught this for me.